pyrft/stats_and_aux_functions.py

```diff
@@ -91,14 +91,14 @@ def contrast_tstats(lat_data, design, contrast_matrix, check_error = 1):
   print(tstat.field.reshape(lat_data.masksize)); print(mvtstat(lat_data.field)[0])
 
   # Two Sample tstat
-  Dim = (10,10); N = 30; categ = np.random.binomial(1, 0.4, size = N)
-  X = group_design(categ); C = np.array((1,-1)); lat_data = pr.wfield(Dim,N)
-  tstats = contrast_tstats(lat_data, X, C)
+Dim = (10,10); N = 30; categ = np.random.binomial(1, 0.4, size = N)

```

Comment:
Please do not use several instructions on the same line. It is terrible for readability.

Reply:
This is just within the examples section, does it matter within that? I wanted to avoid making the docstring enormous.

Reply:
It make code non-readable, which is bad. one more line comes at no cost !

Reply:
But it would be 7 lines instead of 4 right? It makes it more difficult to copy paste which what you need to do with the examples.